app/models/linkset.rb

```diff
@@ -12,6 +12,14 @@ def empty?
   end
 
   def update_attributes_from_gem_specification!(spec)
-    self.update_attributes!(:home => spec.homepage)
+    meta = spec.metadata || {}
+    self.update_attributes!({
+      :home => spec.homepage,
+      :wiki => meta["wiki_uri"],
+      :docs => meta["documentation_uri"],
+      :mail => meta["mailing_list_uri"],
+      :code => meta["source_code_uri"],
+      :bugs => meta["bug_tracker_uri"]
+    }.delete_if { |_n, v| v.nil? })
```

Comment:
Is there a corresponding PR in rubygems to add these keys as recommended? If think those are fine metadata names.  Is Discourse a mailing list?  What about IRC?  Twitter? Contact Email? Something Freeform?